eng/dependencies.props

```diff
@@ -34,8 +33,6 @@
   <PropertyGroup>
     <CoreFxExpectedPrerelease>preview.18619.1</CoreFxExpectedPrerelease>
```

Comment:
@wtgodbe should we still have a reference to corefx in here? or should it be in the new system?

Reply:
and CoreFxCurrentRef

Reply:
I never understood what that is for?